samples/logging/mikroe_click.overlay

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
This overlay should already be available. No need to copy paste into this folder

Reply:
I think I was working with my own overlay that defined more of the shield. I'll double-check and update.

Reply:
What the app has is the same as the shields/ folder, but I have tried a few ways of compiling while adding our custom shield and I can't get that to work; it always prompts for the shields in Zephyr (if I set `BOARD_ROOT` TO `CMAKE_CURRENT_SOURCE_DIR` it warns there isn't a `boards` folder) and says ours doesn't exist.

I think the permanent solution would be to set an environment variable to the user's FSW root and add appropriate calls to all of our CMakeLists files